README.md

```diff
@@ -0,0 +1,22 @@
+# Filter Side Effects
+
+`add_filter_side_effect()` allows attaching a callback function to a WordPress filter without requiring that the callback function return the filtered value, use the filtered value, or even return any value. It wraps `add_filter()` and accepts the same arguments with the same defaults.
```

Comment:
"... without requiring that the callback function return the filtered value" makes it sound optional. I think it would be helpful to explicitly state that any return value is dismissed.